### DISCUSSION TITLE
Histo for a specific scene/layer?

### DISCUSSION THREAD

**Discussion by blue-j** [ImageMagick/ImageMagick#4898] [General] [answered]
I see in verbose output that we get a histogram and entropy and other nice measurements for each layer in a PSD.  Is it possible to target that data more granularly, and just extract the histo data for a specific scene/layer?

Thanks in advance for your attention!

- J

**Comment by snibgo** [comment]
Insert [0] immediately after the input filename (no spaces) to process just the first image, or [1] for the second, etc.

**Comment by fmw42** [comment, marked as answer]
`magick identify -verbose image.psd[X]` where X is the layer as Imagemagick sees them. They might not correspond exactly to the PSD layers if those layers are not simple layers.
